src/test/rpc/LedgerData_test.cpp

```diff
@@ -314,6 +314,34 @@ class LedgerData_test : public beast::unit_test::suite
         auto const USD = gw["USD"];
         env.fund(XRP(100000), gw);
 
+        auto makeRequest = [&env](Json::StaticString const& type) {
```

Comment:
I haven't seen this syntax before. I see it's creating a function, but what does `[&env]` do?

Reply:
https://en.cppreference.com/w/cpp/language/lambda

Reply:
`[&env]` captures the variable `env` by reference.